src/ReactiveUI.Maui/ActivationForViewFetcher.cs

```diff
@@ -55,9 +66,10 @@ public IObservable<bool> GetActivationForView(IActivatableView view)
         return activation.DistinctUntilChanged();
     }
 
-    private static IObservable<bool>? GetActivationFor(ICanActivate? canActivate) => canActivate?.Activated.Select(_ => true).Merge(canActivate.Deactivated.Select(_ => false));
+    private static IObservable<bool>? GetActivationFor(ICanActivate? canActivate) =>
+        canActivate?.Activated.Select(_ => true).Merge(canActivate.Deactivated.Select(_ => false));
 
-#if HAS_MAUI
+#if HAS_MAUI || (HAS_WINUI && IS_MAUI)
```

Comment:
What's the difference between HAS_MAUI and IS_MAUI? Should we rename these?

Reply:
These are to distinguish the Maui Android / iOS section (HAS_MAUI) and the Maui WinUI section HAS_WinUI, as we are using a single file between the two projects (Maui and WinUI) the namespaces need to match the assembly hence the IS_MAUI and IS_WINUI relates to the Project SDK values of UseMaui and UseWinUI.

Reply:
Is there a way for us to have more meaningful name for one of them. Don't mind a bit of verbosity.

Reply:
I can look at dropping HAS_MAUI and invert HAS_WINUI where required, perhaps make HAS_WINUI => WINUI_TARGET

Reply:
Yeah seems nicer name.